hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/fee/SmartContractFeeBuilderTest.java

```diff
@@ -50,4 +52,15 @@ void assertGetFileContentQueryFeeMatrices() {
         assertEquals(0, result.getServicedata().getSbh());
         assertEquals(0, result.getServicedata().getTv());
     }
+
+    @Test
```

Comment:
1. Why not test the two query ones as well (`getContractCallLocalFeeMatrices`, `getContractByteCodeQueryFeeMatrices`)?

1. Needs a comment explaining why these `xxxFeeMatrices` methods need this specific test, it's not really obvious.

Reply:
2. I've added the clarifying comment

1. I haven't done tests for them because the issue describes that we should "harden" only calculateFees() methods. (However, I could add some tests if you think it would be useful.)

I think that the removal of pureChecks from IngestFlow won't affect the queries and computeFees() methods, so it's "hardening" the computeFees() is probably not related to the issue. Maybe @netopyr could give some clarification about this.